tools/CouchDB_Import_Demographics.php

```diff
@@ -123,8 +127,36 @@ function _getProject($id) {
 
     function _generateQuery() {
         $config = NDB_Config::singleton();
-        $fieldsInQuery = "SELECT c.DoB, c.CandID, c.PSCID, s.Visit_label, s.SubprojectID, p.Alias as Site, c.Gender, s.Current_stage, CASE WHEN s.Visit='Failure' THEN 'Failure' WHEN s.Screening='Failure' THEN 'Failure' WHEN s.Visit='Withdrawal' THEN 'Withdrawal' WHEN s.Screening='Withdrawal' THEN 'Withdrawal' ELSE 'Neither' END as Failure, c.ProjectID, c.flagged_caveatemptor as CEF, c.flagged_caveatemptor as CEF, c_o.Description as CEF_reason, c.flagged_other as CEF_comment, pc_comment.Value as Comment, COALESCE(pso.Description,'Active') as Status, ps.participant_suboptions as Status_reason, ps.reason_specify as Status_comments, ps.study_consent as Study_consent, COALESCE(ps.study_consent_withdrawal,'0000-00-00') AS Study_consent_withdrawal";
-        $tablesToJoin = " FROM session s JOIN candidate c USING (CandID) LEFT JOIN psc p ON (p.CenterID=s.CenterID) LEFT JOIN caveat_options c_o ON (c_o.ID=c.flagged_reason) LEFT JOIN parameter_candidate AS pc_comment ON (pc_comment.CandID=c.CandID) AND pc_comment.ParameterTypeID=(SELECT ParameterTypeID FROM parameter_type WHERE Name='candidate_comment') LEFT JOIN participant_status ps ON (ps.CandID=c.CandID) LEFT JOIN participant_status_options pso ON (pso.ID=ps.participant_status)";
+
+        $fieldsInQuery = "SELECT c.DoB,
+                                c.CandID, 
+                                c.PSCID, 
+                                s.Visit_label, 
+                                s.SubprojectID, 
+                                p.Alias as Site, 
+                                c.Gender, 
+                                s.Current_stage, 
+                                CASE WHEN s.Visit='Failure' THEN 'Failure' WHEN s.Screening='Failure' THEN 'Failure' WHEN s.Visit='Withdrawal' THEN 'Withdrawal' WHEN s.Screening='Withdrawal' THEN 'Withdrawal' ELSE 'Neither' END as Failure, 
+                                c.ProjectID, 
+                                c.flagged_caveatemptor as CEF, 
+                                c_o.Description as CEF_reason, 
+                                c.flagged_other as CEF_comment, 
+                                pc_comment.Value as Comment, 
+                                COALESCE(pso.Description,'Active') as Status, 
+                                ps.participant_suboptions as Status_reason, 
+                                ps.reason_specify as Status_comments, 
+                                ps.study_consent as Study_consent, 
+                                COALESCE(ps.study_consent_withdrawal,'0000-00-00') AS Study_consent_withdrawal,
```

Comment:
@xlecours it seems to be on purpose here... otherwise the coalesce would be useless.